docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,67 @@
+# Release 403 (15 Nov 2022)
+
+## General
+
+* Add the amount of data read from external sources during table scans to
+  `EXPLAIN ANALYZE`. ({issue}`14907`)
+* Override the values of the `exchange.compression-enabled`,
+  `query.low-memory-killer.delay`, `query.remote-task.max-error-duration`, and
+  `task.concurrency` configuration properties to the recommended values for
+  fault-tolerant execution when it is enabled. ({issue}`14957`)
+* Improve performance of [exchange management](fte-exchange-manager) encryption
+  in fault-tolerant execution. ({issue}`14941`)
+* Fix potential failure when clients do not support variable precision temporal
+  types. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
+  function. ({issue}`14886`)
+* Fix failure for certain queries involving joins over partitioned tables. ({issue}`14317`)
+* Fix incorrect order of parameters in `DESCRIBE INPUT` when they appear in a
+  `WITH` clause. ({issue}`14738`)
+
+## Security
+
+* Allow access token passthrough when using OAuth 2.0 with refresh tokens
+  enabled. ({issue}`14949`)
+
+## BigQuery connector
+
+* Improve performance of `SHOW SCHEMAS` by adding a metadata cache. This can be
+  configured with the `bigquery.metadata.cache-ttl` catalog property, which is
+  disabled by default. ({issue}`14729`)
+* Fix failure when a [row access 
+  policy](https://cloud.google.com/bigquery/docs/row-level-security-intro)
+  returns an empty result. ({issue}`14760`)
+
+## ClickHouse connector
+
+* Add support for ClickHouse `DateTime(timezone)` type in read-only operations. ({issue}`13541`)
+
+## Hive connector
+
+* Don't show the unsupported `AUTHORIZATION ROLE` property in the result of
+  `SHOW CREATE SCHEMA` if the access control doesn't support roles. ({issue}`8817`)
+
+## Iceberg connector
+
+* Improve performance and reduce storage used by the `expire_snapshots` table
+  procedure on S3-compatible storage. ({issue}`14434`)
+* Allow registering existing table files in the metastore with the new
+  [`register_table` procedure](iceberg-register-table). ({issue}`13552`)
+
+## MongoDB connector
+
+* Add support for {doc}`/sql/delete`. ({issue}`14864`)
+* Fix incorrect results for predicate pushdown applied to columns with `varchar`
+  and `char` types when MongoDB collections have a collation. ({issue}`14900`)
+
+## SQL Server connector
+
+* Fix incorrect results when non-transactional `INSERT` is disabled and bulk
```

Comment:
results of what ... the INSERT query or something else ?? Did it use to fail entirely or insert wrong data or something else?

Reply:
> results of what ... the INSERT query or something else ?? 

Results of `INSERT`

> Did it use to fail entirely or insert wrong data or something else?

Inserted wrong data.

Reply:
When bulk insert is enabled, the copy of the table that Trino loads ends up being empty. I think this is probably the most succinct way of explaining it without getting too into the weeds?

Reply:
Please note that the wrong data were all null, not empty.